exchanges/btse/btse_wrapper.go

```diff
@@ -714,10 +785,110 @@ func (b *BTSE) ValidateCredentials() error {
 
 // GetHistoricCandles returns candles between a time period for a set time interval
 func (b *BTSE) GetHistoricCandles(pair currency.Pair, a asset.Item, start, end time.Time, interval kline.Interval) (kline.Item, error) {
-	return kline.Item{}, common.ErrFunctionNotSupported
+	if err := b.ValidateKline(pair, a, interval); err != nil {
+		return kline.Item{}, err
+	}
+
+	fpair, err := b.FormatExchangeCurrency(pair, a)
+	if err != nil {
+		return kline.Item{}, err
+	}
+	intervalInt, err := strconv.Atoi(b.FormatExchangeKlineInterval(interval))
+	if err != nil {
+		return kline.Item{}, err
+	}
+
+	klineRet := kline.Item{
+		Exchange: b.Name,
+		Pair:     pair,
+		Asset:    a,
+		Interval: interval,
+	}
+
+	switch a {
+	case asset.Spot:
+		req, err := b.OHLCV(fpair.String(),
+			start,
+			end,
+			intervalInt)
+		if err != nil {
+			return kline.Item{}, err
+		}
+		for x := range req {
+			timeStamp, err := convert.TimeFromUnixTimestampFloat(req[x][0])
```

Comment:
In this context, there's a happy little bug which will set all the candles to 1970~. Can use time.Unix(int64(req[x][0]), 0) instead. Same for extended candle handling too